src/lib/eina/eina_sched.c

```diff
@@ -17,68 +17,22 @@
  */
 
 #ifdef HAVE_CONFIG_H
-# include "config.h"
+#include "config.h"
 #endif
+#include "eina_thread.h"
 
-#include <pthread.h>
 #ifdef __linux__
-# include <sched.h>
-# include <sys/time.h>
-# include <sys/resource.h>
-# include <errno.h>
+#include <sched.h>
+#include <sys/time.h>
+#include <sys/resource.h>
+#include <errno.h>
 #endif
 
 #include "eina_sched.h"
 #include "eina_log.h"
 
-#define RTNICENESS 1
-#define NICENESS 5
```

Comment:
Why removing this? If you will not do a win32/posix version shouldn't the be in a #ifdef __linux__ ?

Reply:
I put it on eina_thread.h, to use with *_sched_{win32, posix}

Reply:
Now it is on eina_sched.h